data/keystore/build.gradle

```diff
@@ -0,0 +1,21 @@
+jar {
+    baseName 'teku-data-keystore'
+    manifest {
+        attributes('Implementation-Title': baseName,
+                'Implementation-Version': project.version)
+    }
+}
+
```

Comment:
We shouldn't specify any of this.  The jar config is handled by the top level gradle build file and setup automatically for every sub module.

Reply:
gone